frontend/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment } from 'react';
+import './App.css';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { Products, SingleProduct, WishList, Landing, Cart } from './screens';
+import { Navbar } from './components'
+
+const App = () => {
+  return (
+    <div className="app">
+      <Router>
+        <Switch>
```

Comment:
This led me to read more about <Switch /> and I will be adding this to my project too , good benifit of using switch here

Reply:
yes